### ISSUE TITLE
Custom fields lose connection to their section if custom sections are renamed

### ISSUE THREAD

**Issue by AnotherWayIn** [pwndoc/pwndoc#261]
It would be nice if the fields stay associated with their sections when sections are renamed. This would save having to write up the entire field again

thanks

**Comment by yeln4ts** [comment]
You are right I will look into that as it also concerns Audit Types sections
